dev/ProgressBar/ProgressBar.h

```diff
@@ -50,6 +51,9 @@ class ProgressBar :
     static constexpr wstring_view s_IndeterminateProgressBarIndicator2Name{ L"IndeterminateProgressBarIndicator2" };
     static constexpr wstring_view s_ErrorStateName{ L"Error" };
     static constexpr wstring_view s_PausedStateName{ L"Paused" };
+    static constexpr wstring_view s_CollapsedStateName{ L"Collapsed" };
+    static constexpr wstring_view s_CollapsedErrorStateName{ L"CollapsedError" };
+    static constexpr wstring_view s_CollapsedPausedStateName{ L"CollapsedPaused" };
```

Comment:
Why are these extra states needed?

Reply:
The issue is that if we switch from "normal" state to error or paused, users would see animations that animate to that color. To mitigate this, I added these states such that in the case of a determinate progressbar, users see the control directly as it should be and not have it transition the color again.

Reply:
Just tested it locally. If we switch the visual states to the corresponding determinate versions of error or paused, it does not cause the colour transition.

Reply:
Hope you don't mind me going ahead and making changes to your PR :)

Reply:
Not at all, totally forgot this PR existed. Thank you @karkarl for making the necessary modifications!